src/Spanner/Connection/Grpc.php

```diff
@@ -445,6 +449,38 @@ public function deleteSession(array $args)
     }
 
     /**
+     * Note: This should be removed once GAPIC exposes the ability to execute
+     * concurrent requests.
+     *
+     * @access private
+     * @param array $args
+     * @return UnaryCall
+     * @experimental
+     */
+    public function deleteSessionAsync(array $args)
+    {
+        $database = $this->pluck('database', $args);
+        $headerDescriptor = new AgentHeaderDescriptor([
```

Comment:
Could this be created in the constructor, to avoid the `file_get_contents` call on every request?